mavros_extras/src/plugins/tunnel.cpp

```diff
@@ -30,26 +31,49 @@ class TunnelPlugin : public plugin::PluginBase
 
     void ros_callback(const mavros_msgs::Tunnel::ConstPtr& ros_tunnel)
     {
-        const auto mav_tunnel =
-            copy_tunnel<mavros_msgs::Tunnel, mavlink::common::msg::TUNNEL>(
-                *ros_tunnel);
+        try
+        {
+            const auto mav_tunnel =
+                copy_tunnel<mavros_msgs::Tunnel, mavlink::common::msg::TUNNEL>(
+                    *ros_tunnel);
 
-        UAS_FCU(m_uas)->send_message_ignore_drop(mav_tunnel);
+            UAS_FCU(m_uas)->send_message_ignore_drop(mav_tunnel);
+        }
+        catch (const std::overflow_error& e)
+        {
+            ROS_ERROR_STREAM_NAMED("tunnel", e.what());
+        }
```

Comment:
You wouldn't get overflow exception because `_ignore_drop` variation of `send_message()` catch it.